tests/datasets/test_ogb_graph.py

```diff
@@ -0,0 +1,8 @@
+from gammagl.datasets.ogb_graph import OgbGraphDataset
+
+def test():
+    data=OgbGraphDataset('ogbg-molhiv')
+    print(data)
+    print(data[0])
+
+test()
```

Comment:
Replace the function name "test" with "test_ogbgraphdataset", which makes it easier to distinguish the functions from other files.